### DISCUSSION TITLE
Make Cloudflare Optional

### DISCUSSION THREAD

**Discussion by mrueg** [onedr0p/cluster-template#920] [Ideas]
It would be nice to make cloudflare optional (e.g. move it to the addons) and allow for different solutions like https://tailscale.com/kb/1223/tailscale-funnel/

**Comment by onedr0p** [comment]
I'm with you on this but not sure if/when I'll have time to tackle it.

**Reply by mrueg** [reply]
Maybe as a first step, we could figure out how to support different DNS providers on external-dns and then secondly look at cloudflare tunnel
